torch/csrc/jit/script/compiler.cpp

```diff
@@ -1288,8 +1292,8 @@ struct to_ir {
 
       // current_element_assigner uses an induction variable
       // to set a current element
-      if (current_element_assigner) {
-        current_element_assigner(trip_count, environment_stack);
+      if (iter_val != nullptr && targets) {
```

Comment:
The comment above is now nonsense, and I can't follow what iter_val or targets are in this context.